test/data/getAccount_0x435d7d41d4f69f958bda7a8d9f549a0dd9b64c86.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
can we make this a symbolic to `test/data/getAccount_0.0.4436613.json`? These are the same file/account right? There are other accounts using this approach.